jetty-rewrite/src/main/config/etc/jetty-rewrite-customizer.xml

```diff
@@ -9,7 +9,7 @@
       <New id="Rewrite" class="org.eclipse.jetty.rewrite.RewriteCustomizer">
         <Set name="rewriteRequestURI" property="jetty.rewrite.rewriteRequestURI"/>
         <Set name="rewritePathInfo" property="jetty.rewrite.rewritePathInfo"/>
-        <Set name="originalPathAttribute" property="jetty.rewrite.originalPathAttribute"/>
+        <Set name="originalPathAttribute"><Property name="jetty.rewrite.originalPathAttribute" default="requestedPath"/></Set>
```

Comment:
Seems like a breaking change?

Reply:
It was this way before the commit that changed the xml that kicked off this PR: https://github.com/eclipse/jetty.project/commit/343cf732b9a5aacd4c84fcb937a4b1624a1a0afd

Reply:
I think in this case  we can make this one a `property` as I see no reason for the embedded default to be different from the deployed default.

Reply:
So to be crystal clear on this:   `RewriteCustomer` has a default of `null` for `_originalPathAttribute`. The `rewrite-customizer.mod` file has a _commented out_ value for `jetty.rewrite.originalPathAttribute=requestedPath`. Before commit https://github.com/eclipse/jetty.project/commit/343cf732b9a5aacd4c84fcb937a4b1624a1a0afd the xml defaulted it to `requestedPath`. So this PR is simply putting things back the way they were. @gregw and @sbordet do either of you believe the value should now be defaulted _differently_???? If so, to what?

Reply:
This should be restored to the same behavior it had for all the Jetty 10 versions.
The default value of `null` means to not store the originalPath as a request attribute, which is a valid behavior and has been so for all Jetty 10 versions.